quirklike/Assets/AI/FSM/States/Wanderer Specific/IdleState.cs

```diff
@@ -0,0 +1,33 @@
+using UnityEngine;
+using UnityEngine.AI;
+
+public class IdleState : State
+{
+    private readonly Wanderer wanderer;
+    private readonly NavMeshAgent NavAgent;
+    private int IDLE_TIME = 1000;
+    
+    public IdleState(Wanderer w, NavMeshAgent a){
+        this.wanderer = w;
+        this.NavAgent = a;
+    }
+
+    public void Tick()
+    { 
+        IDLE_TIME --;
```

Comment:
Won't this be framerate dependent? 1 tick = 1 Update() so this should probably be a float instead and use deltaTime?